module/nvpair/nvpair.c

```diff
@@ -25,6 +25,12 @@
  * Copyright 2018 RackTop Systems.
  */
 
+/*
+ * Links to Illumos.org for more information on Interface Libraries:
+ * [1] https://illumos.org/man/3lib/libnvpair
```

Comment:
Might link to the kernel versions of these instead (or additionally):
https://illumos.org/man/9f/nvlist_alloc
https://illumos.org/man/9f/nvlist_next_nvpair
https://illumos.org/man/9f/nvpair_value_byte

Reply:
Good point.  Yes, we should link to the kernel versions.

Reply:
@freqlabs and @behlendorf 
Should I add above three links in "module/nvpair/nvpair.c " as well

Reply:
Yes, please do.   Similarly we should add `https://illumos.org/man/9f/kstat_create` to the `spl_kstat.c` files.

Reply:
okay, done